news/2020-01-22-community-choice-2019-voting-open.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Community Choice 2019: Voting Open"
+date: 2020-01-22 10:00:00 +0000
+---
+
+2019 has come and gone, and with it, an entire year of mapping legacy enters the history books forever more. Become a part of history and help us decide which of 2019's maps were the Community Choice!
+
+![](/wiki/shared/news/2020-01-22-community-choice-2019-voting-open/header.jpg)
+
+In case you're unfamiliar, every year we run a mass poll including all Ranked maps submitted across a calendar year to help determine which maps were the most adored by the community at large. This has gone by a variety of names, from the venerable "Best Of 20XX" series back in the earlier days of the game's history, to the more recent "Community Choice" series.
+
+Each and every player gets **10 votes per game mode** to cast for **any Ranked map from 2019 which they have played at least once**. 
+
+You don't have to worry about figuring all this out - the power of our contest system will auto-magically handle this for you, and present you with a list of all the entries you're eligible to vote for.
+
+Voting is super simple: just pick through your list for the maps that wowed you the most, and give them a star with a click.
+
+The highest voted maps from each mode will be branded as the Community Favourites of 2019, and net their authors a **shiny new profile badge**, plus **6 months of osu!supporter**. The other maps in the top 10 will all see their authors receive **3 months of osu!supporter** to honour their efforts.
+
+Got all that? When you're ready to vote, head to the following links below:
+
+* [osu!](https://osu.ppy.sh/community/contests/85)
+* [osu!taiko](https://osu.ppy.sh/community/contests/86)
+* [osu!catch](https://osu.ppy.sh/community/contests/87)
+* [osu!mania](https://osu.ppy.sh/community/contests/88)
+
+The polls will remain open until the **6th of February, 2020**, which is exactly **15 days** from the date of this post. Get in and get your votes done quick, you'll be surprised how fast it passes by.
+
+Good luck!
```

Comment:
"good luck" for voting? the last paragraph seems like a good place to end the post regardless, i dont think  u need another short message like this